### DISCUSSION TITLE
延时分布监控的数据字段说明文档

### DISCUSSION THREAD

**Discussion by lf2186** [bfenetworks/bfe#844] [Q&A (问题和回答)] [answered]
hi，
https://www.bfe-networks.net/zh_cn/monitor/latency/

$ curl -s 'http://localhost:8421/monitor/proxy_handshake_delay?format=prometheus'
# TYPE proxy_handshake_delay_Past histogram
proxy_handshake_delay_Past_bucket{le="1000"} 8065
proxy_handshake_delay_Past_bucket{le="2000"} 9924
proxy_handshake_delay_Past_bucket{le="3000"} 10278
proxy_handshake_delay_Past_bucket{le="4000"} 10529
proxy_handshake_delay_Past_bucket{le="5000"} 10683
proxy_handshake_delay_Past_bucket{le="6000"} 10785
proxy_handshake_delay_Past_bucket{le="7000"} 10859
proxy_handshake_delay_Past_bucket{le="8000"} 10934
proxy_handshake_delay_Past_bucket{le="9000"} 10982
proxy_handshake_delay_Past_bucket{le="10000"} 11026
proxy_handshake_delay_Past_bucket{le="+Inf"} 11466
proxy_handshake_delay_Past_sum 3160545584
proxy_handshake_delay_Past_count 11466

请问什么时候，可以补充一下这些key的详细说明文档，包括key含义 和 时间粒度等等

**Comment by iyangsj** [comment, marked as answer]
@lf2186 这些字段代表了TLS握手延迟的分布情况的统计，数据格式含义可参考 https://prometheus.io/docs/concepts/metric_types/#histogram

示例说明如下：
- proxy_handshake_delay_Past_bucket{le="1000"} 代表握手延迟小于1000ms 的握手次数
- proxy_handshake_delay_Past_bucket{le="2000"}代表握手延迟小于2000ms之间的握手次数，依次类推
- proxy_handshake_delay_Past_bucket{le="+Inf"} 代表代表握手延迟小于无穷大的握手次数（相当于总握手此数）
- proxy_handshake_delay_Past_sum/proxy_handshake_delay_Past_count 分别代表总握手延迟及握手次数

下个版本发布的文档将包含详细说明
